### PR TITLE
Fixes #1835 logical definition child terms of 'brush cell of tracheobronchial tree'

### LINKED ISSUE #1835: [Logical def] Revise logical definition of child terms of 'brush cell of trachebronchial tree'
**CL term**
CL:0002208
CL:0002206
CL:0002075

**Suggested revision of logical definition**
Correcting the genus from 'brush cell of trachebronchial tree' to 'brush cell'. Example with 'brush cell of bronchus':

_Current logical definition_
[brush cell of trachebronchial tree](https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0002075) and [part of](https://www.ebi.ac.uk/ols/ontologies/cl/properties?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000050) some [epithelium of bronchus](https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0002031)

_New logical definition_
[brush cell](https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0002204) and [part of](https://www.ebi.ac.uk/ols/ontologies/cl/properties?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000050) some [epithelium of bronchus](https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0002031)

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #1835 logical definition 'brush cell of bronchus'

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -15507,7 +15507,9 @@ AnnotationAssertion(oboInOwl:creation_date obo:CL_0002205 "2010-09-02T02:38:34Z"
 AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_0002205 "CL:1000386"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002205 "FMA:263220"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002205 "brush cell of lobular bronchiole"^^xsd:string)
-EquivalentClasses(obo:CL_0002205 ObjectIntersectionOf(obo:CL_0002075 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002051)))
+EquivalentClasses(obo:CL_0002205 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002051)))
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:15510]: Based on the label 'brush cell of lobular bronchiole', this EA may be too broad.

The label suggests this cell type is part of the epithelium of a 'lobular bronchiole'.
The EA asserts this cell type is part of 'epithelium of bronchiole', which suggests other distal structures such as "terminal bronchiole" are included.

This seems compounded by UBERON stating 'epithelium of bronchiole' has exact synonym 'epithelium of lobular bronchiole", which seems to stem from FMA2 and FMA3.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -15507,7 +15507,9 @@ AnnotationAssertion(oboInOwl:creation_date obo:CL_0002205 "2010-09-02T02:38:34Z"
...
+SubClassOf(obo:CL_0002205 obo:CL_0002075)
+SubClassOf(obo:CL_0002205 obo:CL_0002208)
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:15512]: 'bronchial epithelial cell' is defined as an epithelial cell that is part of bronchus. Bronchus and bronchiole are different anatomical structures, so consider removing this.

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -15517,7 +15519,7 @@ AnnotationAssertion(oboInOwl:creation_date obo:CL_0002206 "2010-09-02T02:38:36Z"
 AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_0002206 "CL:1000390"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002206 "FMA:263228"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002206 "brush cell of terminal bronchiole"^^xsd:string)
-EquivalentClasses(obo:CL_0002206 ObjectIntersectionOf(obo:CL_0002075 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001958)))
+EquivalentClasses(obo:CL_0002206 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001958)))
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:15522]: 'brush cell of lobular bronchiole' is an inferred parent class.

My understanding is that lobular bronchiole and terminal bronchiole are distinct anatomical structures, so this inference does not seem correct.

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
See inline comments.

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:15512]: Thank you for the comment, I will eliminate it

Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:15512]: 'brush cell of bronchus' is still an inferred parent class. The reason being that 'bronchiole' is 'branching part of' 'bronchus', and 'branching part of' is a subproperty of part of. 
Screenshot 2023-02-06 at 14 38 26

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:15522]: The inference comes from 'epithelium of bronchiole' is an exact synonym of 'epithelium of lobular bronchiole'
Screenshot 2023-02-06 at 14 51 39

My suggestion is:

- [ ] Create a 'lobular bronchiole epithelium' term separate from ''epithelium of bronchiole''

Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:15522]: This seems reasonable if the synonyms are also removed and 'lobular bronchiole' becomes a new Uberon class.

Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:15522]: I don't think annotated synonyms impact inferences.

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:15512]: Definition of 'branching part of': x is a branching part of y if and only if x is part of y and x is connected directly or indirectly to the main stem of y

A bronchiole is not part of a bronchus (they are separate), so I don't think that property is correctly applied in this case. Asserting that a bronchiole is a 'branching apart of' a 'tracheobronchial tree' seems more appropriate.

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:15522]: 'brush cell of lobular bronchiole' should be `'brush cell' and ('part of' some 'epithelium of lobular bronchiole')`. However, as 'epithelium of lobular bronchiole' is an exact synonym of 'epithelium of bronchiole', it is inferred that 'brush cell of terminal bronchiole' is a subclass of 'brush cell of lobular bronchiole'

**Comment by dosumis** [comment]
#gogoeditdiff

**Comment by dosumis** [comment]
We need to be able to rapidly refactor in this way.  We should always trigger reasoned diffs when we do and judge the edits based on inference.

**Comment by dosumis** [comment]
@anitacaron - this diff is broken: 

> Here's a diff of how these changes impact the classified ontology (on -simple file)

Compare to unreasoned above.  The changes on -simple are unrelated to the diff.

**Comment by anitacaron** [comment]
We just had an update on the GH Action. I'll trigger it again to check.

#gogoeditdiff

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:15522]: bronchiole and lobular bronchiole have been used as synonyms in seven terms (some are repeated):
Screenshot 2023-02-06 at 16 15 02

Therefore, I will expand the creation of new terms.

**Comment by anitacaron** [comment]
#gogoeditdiff

**Comment by anitacaron** [comment]
#gogoeditdiff

**Comment by bvarner-ebi** [comment]
@aleixpuigb, checking in on this PR.
Kindly advise if there are pending questions.

Of note, the label for CL:0002075 (and in the title of this PR and associated ticket) appears to have a typo:

'brush cell of trachebronchial tree'

should be:

'brush cell of tracheobronchial tree'

**Comment by aleixpuigb** [comment]
> @aleixpuigb, checking in on this PR.
> Kindly advise if there are pending questions.

Thank you @bvarner-ebi. No need for further reviews yet, as I am [creating new terms for UBERON](https://github.com/obophenotype/uberon/issues/2795). Next, I will edit be able to fix the unwanted inference.

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**